plugins/figma/src/common/utils.ts

```diff
@@ -0,0 +1,82 @@
+import type { ThemeInfo } from '@digdir/designsystemet/color';
+
+import type { ThemeInfo as Test } from './store';
+
+export const hexToRgb = (hex: string) => {
+  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+  return result
+    ? {
+        r: parseInt(result[1], 16) / 255,
+        g: parseInt(result[2], 16) / 255,
+        b: parseInt(result[3], 16) / 255,
+      }
+    : null;
+};
+
+export const rgbToHex = (rgb: { r: number; g: number; b: number }) => {
+  return (
+    '#' +
+    [rgb.r, rgb.g, rgb.b]
+      .map((x) => {
+        const hex = Math.round(x * 255).toString(16);
+        return hex.length === 1 ? '0' + hex : hex;
+      })
+      .join('')
+  );
+};
```

Comment:
These utils functions should be used from `@digdir/designsystemet/color` instead of duplicate decleration.

Reply:
Moved